packages/backend/tools/bootstrap-modos/README.md

```diff
@@ -0,0 +1,82 @@
+# Bootstrap-modos
```

Comment:
Don't you think it would be better to create a new package into the monorepo for tools like that ? I'm not sure but what do you think ?